t/30-mmapi.t

```diff
@@ -15,6 +15,7 @@ BEGIN {
     $ENV{OS_AUTOINST_MMAPI_RETRY_COUNT} = 1;
     $ENV{OS_AUTOINST_MMAPI_RETRY_INTERVAL} = 0;
     $ENV{OS_AUTOINST_MMAPI_POLL_INTERVAL} = 0;
+    $ENV{OS_AUTOINST_LOCKAPI_POLL_INTERVAL} = 1;
```

Comment:
It would make sense to use `Test::MockTime` here so the `sleep` doesn't actually sleep. (This module will cause `time` to report the slept time but not actually sleep.)

Of course one could also mock the sleeping manually or just set the interval to zero and the timeout to zero.

Reply:
I second that. https://github.com/os-autoinst/os-autoinst/actions/runs/5467368921/jobs/9953579888#step:3:679 from the latest commit on master shows a test runtime of 2s vs. https://github.com/os-autoinst/os-autoinst/actions/runs/5466579991/jobs/9951710094?pr=2337#step:3:679 for your PR. That's a significant and too big increase. I suggest you crosscheck the runtime of t/30-mmapi.t locally and ensure that it doesn't get too high.

Reply:
https://metacpan.org/pod/Test::MockSleep is what we would need, or? I think `Test::MockTime` can set the time value, but we do a sleep which cause the delay. And with
```perl
use Test::MockSleep qw(:with_mocktime);
```
it would be solved.

But I don't see a package for it. How do we install the dependencies for os-autoinst tests?

Reply:
I was under the impression that `use Test::MockTime qw( :all );` would do the trick. The dependency would likely go under https://github.com/os-autoinst/os-autoinst/blob/d13a461849f1d8fe027f64008f18fe48d1c576fa/dependencies.yaml#L76. You'll have to invoke the build system target `pdate-deps` to apply the change to this file in other places.

Reply:
Ok I think I got it, it's `Test::MockTime` vs `Test::Mock::Time` :facepalm: 

```perl
time perl <<'EOT'
use Mojo::Base -strict;
use Test::Mock::Time;

my ($start) = time;
sleep 2;
my ($end) = time;
say "end:$end - start:$start = " . ($end - $start);
EOT
```